br/pkg/storage/gcs.go

```diff
@@ -97,14 +98,23 @@ func (options *GCSBackendOptions) parseFromFlags(flags *pflag.FlagSet) error {
 // GCSStorage defines some standard operations for BR/Lightning on the GCS storage.
 // It implements the `ExternalStorage` interface.
 type GCSStorage struct {
-	gcs    *backuppb.GCS
-	bucket *storage.BucketHandle
-	cli    *storage.Client
+	gcs       *backuppb.GCS
+	idx       *atomic.Int64
+	clientCnt int64
+	buckets   []*storage.BucketHandle
```

Comment:
```suggestion
	handles   []*storage.BucketHandle
```

Reply:
it's bucket handle and GCS package also have other handles like object handles, so I think the current name is good

Reply:
it reads like we are using multiple `gcs buckets`, but is multiple handles of same bucket actually